pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/MailingAddressComponent.java

```diff
@@ -16,12 +16,15 @@
     private transient Long titleTemplateId;
     private transient Long subtitleTemplateId;
 
-    public MailingAddressComponent(Long titleTemplateId, Long subtitleTemplateId, boolean hideNumber) {
+    public MailingAddressComponent(Long titleTemplateId, Long subtitleTemplateId, boolean hideNumber,
+                                   boolean requireVerified, boolean requirePhone) {
         super(ComponentType.MAILING_ADDRESS);
         hideDisplayNumber = hideNumber;
-        titleTemplateId = titleTemplateId;
-        subtitleTemplateId = subtitleTemplateId;
+        this.titleTemplateId = titleTemplateId;
+        this.subtitleTemplateId = subtitleTemplateId;
```

Comment:
This was a bug! We got overshadowed variables here and need to use `this` qualifier! This resulted in title/subtitle NOT being assigned properly.